From a90857543b653d1bdc946666629fef7850c81e4f Mon Sep 17 00:00:00 2001
From: P Praneesh <quic_ppranees@quicinc.com>
Date: Mon, 3 Jan 2022 11:46:20 +0530
Subject: [PATCH] ath12k: rearrange ring type enum to match the hal ring table

Enum hal_ring_type used to refer the hal ring table entry, mismatch between thes
two leads to fetching wrong ring table information. Align ring type enum to match
with the hal ring table.

Signed-off-by: P Praneesh <quic_ppranees@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/hal.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/ath/ath12k/hal.h b/drivers/net/wireless/ath/ath12k/hal.h
index 3b8181a..5a20df8 100644
--- a/drivers/net/wireless/ath/ath12k/hal.h
+++ b/drivers/net/wireless/ath/ath12k/hal.h
@@ -523,14 +523,14 @@ enum hal_ring_type {
 	HAL_WBM2SW_RELEASE,
 	HAL_RXDMA_BUF,
 	HAL_RXDMA_DST,
-	HAL_RXDMA_MONITOR_BUF,
-	HAL_RXDMA_MONITOR_DST,
 	HAL_RXDMA_MONITOR_DESC,
 	HAL_RXDMA_MONITOR_STATUS,
 	HAL_RXDMA_DIR_BUF,
 	HAL_PPE2TCL,
 	HAL_PPE_RELEASE,
+	HAL_RXDMA_MONITOR_BUF,
 	HAL_TX_MONITOR_BUF,
+	HAL_RXDMA_MONITOR_DST,
 	HAL_TX_MONITOR_DST,
 	HAL_MAX_RING_TYPES,
 };
-- 
2.17.1

